packages/babel-plugin-transform-classes/src/inline-createSuper-helpers.js

```diff
@@ -0,0 +1,71 @@
+import { template, types as t } from "@babel/core";
+
+const helperIDs = new WeakMap();
+
+export default function addCreateSuperHelper(file) {
+  try {
+    return file.addHelper("createSuper");
+  } catch {
+    // Babel <7.9.0 doesn't support the helper.
+  }
+
+  if (helperIDs.has(file)) {
+    // t.cloneNode isn't supported in every version
+    return (t.cloneNode || t.clone)(helperIDs.get(file));
+  }
+
+  const id = file.scope.generateUidIdentifier("createSuper");
+  helperIDs.set(file, id);
+
+  const fn = helper({
+    CREATE_SUPER: id,
+    GET_PROTOTYPE_OF: file.addHelper("getPrototypeOf"),
+    POSSIBLE_CONSTRUCTOR_RETURN: file.addHelper("possibleConstructorReturn"),
+  });
+
+  file.path.unshiftContainer("body", [fn]);
+  file.scope.registerDeclaration(file.path.get("body.0"));
+
+  return t.cloneNode(id);
+}
+
+const helper = template.statement`
```

Comment:
May worth leave a note at `babel-helpers` that we should manually sync these two sources.